dio/lib/src/adapters/browser_adapter.dart

```diff
@@ -181,9 +182,10 @@ class BrowserHttpClientAdapter implements HttpClientAdapter {
         try {
           xhr.abort();
         } catch (_) {}
-        // xhr.onError will not triggered when xhr.abort() called.
-        // so need to manual throw the cancel error to avoid Future hang ups.
-        // or added xhr.onAbort like axios did https://github.com/axios/axios/blob/master/lib/adapters/xhr.js#L102-L111
+        // `xhr.onError` will not triggered when `xhr.abort()` is called.
+        // We need to manually throw the exception to avoid `Future` hangs,
+        // or adding `xhr.onAbort` like axios:
```

Comment:
Instead of writing this as a comment, you should create an issue. Otherwise, it might get forgotten.

Reply:
I think this is already addressed. Removing these